include/nuttx/sched.h

```diff
@@ -771,14 +772,16 @@ begin_packed_struct struct tcbinfo_s
   uint16_t state_off;                    /* Offset of tcb.task_state        */
   uint16_t pri_off;                      /* Offset of tcb.sched_priority    */
   uint16_t name_off;                     /* Offset of tcb.name              */
-  uint16_t reg_num;                      /* Num of regs in tcbinfo.reg_offs */
+  uint16_t regs_off;                     /* Offset of tcb.regs              */
```

Comment:
Can you please update xtensa_tcbinfo.c too?

Reply:
Yes, @zhuyanlinzyl will fix xtensa_tcbinfo.c in next week.

Reply:
Done.